Client/Components/UnloggedUserComponent.razor.css

```diff
@@ -0,0 +1,33 @@
+﻿.content {
+    display: flex;
+    flex-flow: column;
+    align-items: center;
+    padding-left: 5%;
+    padding-right: 5%;
+    text-align: center;
+    border: #3F4D54 solid 1px;
+    border-radius: 25px;
+    background-color: white;
+}
+
+.create-account {
+    background: -webkit-linear-gradient(left,#ED746E, #F7D964);
```

Comment:
Is there a reason why you use -webkit-linear-gradient instead of linear-gradient?

Reply:
linear-gradient doesn't work for me so i am using -webkit-linear-gradient